raster.py

```diff
@@ -460,27 +460,28 @@ def d13C_var_geotiff() -> AmazonGeoTiff:
 }
 
 # Converts a resolution into a Bounds class, storing info about pixel size and raster size.
-def create_bounds_from_res(res_x: int, res_y: int, brazil_bounds: Bounds):
-  # Use brazil_bounds to get the min/max lat and lon. Scale everything else
+def create_bounds_from_res(res_x: int, res_y: int, bounds: Bounds):
```

Comment:
Did you mean to have this parameter as 'base_bounds' like referenced throughout this function?